tests/unit/lang.ts

```diff
@@ -8,104 +8,107 @@ import { Es5Observer, Es7Observer } from 'src/observers/ObjectObserver';
 registerSuite({
 	name: 'lang functions',
 
-	'.copy() invalid arguments': function () {
-		assert.throw(function () {
-			lang.copy({
-				sources: []
-			});
+	'.assign()'(): void {
+		const source = Object.create({ a: 1 }, {
+			b: {
+				enumerable: false,
+				configurable: true,
+				writable: true,
+				value: 2
+			}
 		});
+		source.c = 3;
+
```

Comment:
You appear to be testing for shallow copying in the `mixin` test but not in the `assign` test.